python/pyrogue/_Variable.py

```diff
@@ -633,13 +630,11 @@ def post(self,value):
         This method does not call through parent.writeBlocks(), but rather
         calls on self._block directly.
         """
-        self._log.debug("{}.post({})".format(self, value))
-
         try:
 
             # Set value to block
-            self._block.set(self, value)
-            self._block.startTransaction(rogue.interfaces.memory.Post, False, True, 0, -1)
+            self._set(value)
+            self._block.startTransaction(rim.Post, False, True, self)
```

Comment:
It would be better to include the argument names in these calls with multiple booleans.

Reply:
Historically I have not done that with cpp calls, but it appears you may be able to defined named variables in the cpp interface. I will take a look.

Reply:
Crap, I forgot this was in C++.

In the case of `check` maybe it's better just to require a separate explicit call to `checkTransaction()`. It's more readable that way.

What is the use case for forced writes again?

Reply:
I was wondering that was well. I was tempered to get rid of it.

Reply:
Forced wries occur when the variable ins directly written to. We could detect that when the passed variable is not null

Reply:
Force gets used on global writes, so it does need to be a separate arg.

Reply:
I added comments anytime the startTransaction is called.